user_guide.adoc

```diff
@@ -380,6 +380,21 @@ enums:
     17: udp
 ----
 
+Alternatively, hexadecimal notation can also be used to define an enumeration:
```

Comment:
Totally ok, but I'd also noted that this is a service provided by YAML, not something specific to KS.

Reply:
Some Construct features are Python features, but I would advertise them just the same. Purpose of documentation is to show capabilities, not attribution. =) Just saying.